### ISSUE TITLE
Table comments truncated when exporting to SQL(mysql)

### ISSUE THREAD

**Issue by tchoulihan** [ondras/wwwsqldesigner#209]
I notice some of the table comments are being truncated when exporting to mysql:

`COMMENT 'design team roles (principal, project manager, civil enginee';`

When the xml looks fine: 
```
<comment>design team roles (principal, project manager, civil engineer, landscape engineer, structural engineer, mechanical engineer, plumbing engineer, electrical engineer, fire protection engineer, lab consultant, geotechnical engineer) and company roles (owner, developer, tenant, general contractor)</comment>
```

**Comment by ondras** [comment]
Right, this is probably intentional, as there is an explicit substring extraction in the relevant template: https://github.com/ondras/wwwsqldesigner/blob/master/db/mysql/output.xsl#L92

The problem is that I truly have no idea **why** do we need to take only the first 60 chars... :smile:

**Comment by tchoulihan** [comment]
Couldn't we take out that substring statement? Want me to submit a PR?

**Comment by ondras** [comment]
I would gladly take it out myself. I just want to know why is the substring present right now...
